<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollAreaSourceImage">
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>239</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblSourceImage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollAreaGrayScaleImage">
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>122</width>
         <height>239</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblGrayScaledImage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollAreaSmoothedImage">
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>140</width>
         <height>314</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblSmoothedImage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollAreaSegmentation">
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>132</width>
         <height>239</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblSegmentation">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="actionLoadImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadImage">
   <property name="text">
    <string>&amp;Загрузить картинку...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
